tutorial/sample-problems/DiffCalc/AnswerWithUnits.pg

```diff
@@ -22,26 +24,25 @@
 #: PODLINK('parserFormulaWithUnits.pl') macros.
 DOCUMENT();
 
-loadMacros(
-    'PGstandard.pl',            'PGML.pl',
-    'parserNumberWithUnits.pl', 'parserFormulaWithUnits.pl',
-    'PGcourse.pl'
-);
+loadMacros('PGstandard.pl', 'PGML.pl', 'contextUnits.pl', 'PGcourse.pl');
 
 #:% section = setup
+#: First, we will set the variable for the functions in the problem to be t.
+#: Since there is a variable of t, we will assign it to be "seconds".
```

Comment:
Change this to something like

```suggestion
#: First, set the variable for the functions in the problem to be t,
#: and assign the unit "seconds" to it.
```

Generally, try to avoid using the "royal we" in the documentation.  That is bad practice.